README.md

```diff
@@ -1,30 +1,60 @@
 
+[![Open in Visual Studio Code](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/microsoft/react-native-test-app)
 ![build](https://github.com/microsoft/react-native-test-app/workflows/build/badge.svg)
-[![npm version](https://badgen.net/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
-
-React Native Test App provides test apps for all platforms as a package. Bring
-your own JS bundle, and let us manage the native bits.
-
-This is a work in progress. You can read and discuss the RFC at
-[React Native: Discussions and Proposals](https://github.com/react-native-community/discussions-and-proposals/pull/204).
-
-For more details, please refer to the
-[wiki](https://github.com/microsoft/react-native-test-app/wiki).
+[![npm version](https://img.shields.io/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
+
+React Native Test App provides test apps for all platforms as a package. It
+handles the native bits for you so you can focus on what's important: Your
+product.
+
+## Motivation
+
+Many of us have been there. We create a new React Native project with
+`react-native init`, write a few test screens for our library, and everything is
+seemingly fine and dandy. Until some time later, when a new version of React
+Native gets published. "Awesome!", you think, but when you try to upgrade to the
+latest version, something doesn't work. You need to look at
+[React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/)
+and comb through all the files that need to be changed. Eventually, you get it
+working and everything is fine again. Then a new version of Xcode or iOS comes
+out, or a new version of Gradle is required, and things stop working again. If
+you're lucky, you get to do this N number of times for all the projects you
+maintain.
+
+`react-native-test-app` aims to take away a lot of these pains.
+
+- We want to make it easy to get started. `react-native-test-app` should provide
+  everything needed to get a cross-platform project set up and running in no
+  time.
+- We want to make it easy to upgrade and downgrade React Native without having
+  to deal with project files for every supported platform. Just change the
+  version numbers in `package.json`, and reinstall the dependencies.
+  - So you can quickly switch between versions of React Native to test
+    compatibility or reproduce difficult bugs.
+  - So your next React Native upgrade is a lot more painless. Especially if you
+    have to repeat the same process in a lot of projects.
+- We want to make it easy to add support for and maintain additional platforms,
+  such as macOS or Windows, without requiring the domain knowledge to do so.
+- We want to give you a consistent developer experience across all the projects
+  you maintain.
+
+You can find the full design document in
+[the wiki](https://github.com/microsoft/react-native-test-app/wiki/Design).
```

Comment:
I like this section, but I don't think many people want to see this as the first thing they read in the README; I would either move it lower down in the document or within the Wiki, and just have an extra line in the first paragraph like:

```md
If you want to learn more about [the motivation](link to wiki) or [the design](link to wiki) of this tool, you can refer to the wiki.
```